packages/circuit-ui/components/TopNavigation/TopNavigation.spec.tsx

```diff
@@ -38,25 +38,29 @@ describe('TopNavigation', () => {
       activeLabel: 'Close menu',
       inactiveLabel: 'Open menu',
     },
-    userName: 'Jane Doe',
-    userId: 'ID: AC3YULT8',
-    profileLabel: 'Open profile menu',
-    profileActions: [
-      {
-        onClick: jest.fn(),
-        children: 'View profile',
-      },
-      {
-        onClick: jest.fn(),
-        children: 'Settings',
-      },
-      { type: 'divider' },
-      {
-        onClick: jest.fn(),
-        children: 'Logout',
-        destructive: true,
-      },
-    ] as PopoverProps['actions'],
+    user: {
+      name: 'Jane Doe',
+      id: 'ID: AC3YULT8',
+    },
+    profileMenu: {
+      label: 'Open profile menu',
+      actions: [
+        {
+          onClick: jest.fn(),
+          children: 'View profile',
```

Comment:
Should we rename this `label` like we did recently in other components, while we're here?

(on the other hand there's already a `profileMenu.label`...)

Reply:
This is using the props from the Popover component, so we would need to change it there. 

Why did we rename `children` to `label` again? The convention in my head was `children` → any React element, `label` → string only.

Reply:
☝️ exactly. Then if this should be a React element, it makes total sense—I thought it should be a string

Reply:
We renamed e.g. the Badge children to label

Reply:
Would need to check the Popover, I think it could be a React child for bold text.